src/app/components/info-card/info-card.tsx

```diff
@@ -126,7 +126,7 @@ export function InfoCardFooter({ children }: InfoCardFooterProps) {
       alignItems="center"
       bg={whenPageMode({
         full: '',
-        popup: '#fff',
+        popup: 'accent.background-primary',
```

Comment:
You're certain this works? Wondered whether the static analysis of panda works when wrapping in the whenPageMode

Reply:
Yes, I did check it.

Reply:
Confirmed, before under the button was actually rendering with no bg color...

<img width="416" alt="Screenshot 2023-11-24 at 11 52 39 AM" src="https://github.com/leather-wallet/extension/assets/6493321/01ef8c5f-9178-4fc2-943f-daa51bf8f85a">

Reply:
Better image to show footer bg is now solid...

<img width="412" alt="Screenshot 2023-11-24 at 12 39 09 PM" src="https://github.com/leather-wallet/extension/assets/6493321/28daee93-56ec-49f9-9ccb-503b2f75f36b">